Add autopep8 config files and format files with autopep8

LuxF3rre opened the pull request:
I added autopep8 config - I set it to split only very long lines > 120 chars.

I ran command `autopep8 --in-place` for all *.py files which resulted in more cleaner code.

LuxF3rre committed:
Merge pull request #1 from persepolisdm/master

code sync

LuxF3rre committed:
added autopep8 config

LuxF3rre committed:
autopep8

LuxF3rre committed:
more autopep8

alireza-amirsamimi commented:
Hello
Your codes couldn't pass [Travis CI building](https://travis-ci.org/persepolisdm/persepolis/builds/571863835?utm_source=github_status&utm_medium=notification) successfully. Please check your codes.

LuxF3rre committed:
possible fix

LuxF3rre committed:
Merge branch 'master' into code-cleanup

LuxF3rre committed:
check if directory exist

LuxF3rre commented:
@alireza-amirsamimi fixed :)

alireza-amirsamimi commented:
Thank you very much :)
a little modification in initialization.py and persepolis.py and osCommands.py and test.py is needed.
I'll edit them tomorrow and then I'll accept the pull request.

alireza-amirsamimi committed:
I reverted some changes in initialization.py 

logger must be imported after config_folder and persepolis_tmp creation.
Data base must be imported after log file creation.

alireza-amirsamimi committed:
Some changes are reverted in osCommands.py

Some changes are reverted in osCommands.py

alireza-amirsamimi committed:
some changes are reverted in test.py

Program must be run after uid check.
User can't run persepolis as root. It's unsafe to run GUI as root.
